src/server.js

```diff
@@ -315,13 +320,15 @@ function ServerMock(httpConfig, httpsConfig)
     var httpServerMock  = httpConfig ?  new Server(httpConfig.host, httpConfig.port) : new ServerVoid();
     var httpsServerMock = httpsConfig ? new Server(httpsConfig.host, httpsConfig.port, httpsConfig.key, httpsConfig.cert) : new ServerVoid();
 
-    this.start = function(callback) {
-        httpServerMock.start(function() {
-            httpsServerMock.start(function() {
+    this.start = (function(callback) {
+        httpServerMock.start((function() {
+            this.httpPort = httpServerMock.server.address().port;
```

Comment:
Instead of exposing the port as a property, what do you think if we add a couple of getter methods `getHttpPort()` and `getHttpsPort()`?

Reply:
Done.